sdk/typescript/docker-compose.yml

```diff
@@ -10,3 +12,15 @@ services:
       POSTGRES_DB: iota_indexer
     ports:
       - "5432:5432"
+    read_only: true
+    tmpfs:
+      - /tmp
+      - /dev/shm
+      - /run
+      - /run/postgresql
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    user: "999:999"
```

Comment:
Why is the user here 999, but in the other change 1001?

Reply:
If I'm not wrong postgres uses this uid and group by default